versionhistory.md

```diff
@@ -0,0 +1,77 @@
+# Version history endpoints
+
+[Back to the list of all defined endpoints](endpoints.md)
+
+This endpoint represents all related versions.
+
+Whether version history information is accessible depends on versioning.item.history.view.admin configuration
+If this is set to true, the version history can only be retrieved if the user is an admin of the last version's item
+
+## Get version history
+
+**GET /api/versioning/versionhistory/<:versionHistoryId>**
```

Comment:
it should be named `/api/versioning/versionhistories/`

Reply:
Good catch, I've adjusted this